core/src/test/scala/org/apache/spark/executor/ClassLoaderIsolationSuite.scala

```diff
@@ -109,4 +109,31 @@ class ClassLoaderIsolationSuite extends SparkFunSuite with LocalSparkContext  {
       }
     }
   }
+
+  test("SPARK-51537 Executor isolation session classloader inherits from " +
```

Comment:
Is there a way we can also add a test (here or elsewhere) for the plugin issue?

Reply:
Could we file a followup for this? For now, I've no idea about how to adding the plugin in this test.

Reply:
Sure!